.github/workflows/on_PR_windows_matrix.yml

```diff
@@ -167,6 +168,7 @@ jobs:
             libexpat-devel
             libiconv-devel
             libinih-devel
+            libfmt-devel
```

Comment:
`https://github.com/Exiv2/exiv2/actions/runs/6856124911/job/18642656996#step:4:90`

Doesn't seem to exist for Cygwin yet, I might request it.

Reply:
Great. I assume temporarily Conan could be used to get fmt.

Reply:
Moved Cygwin to build with C++20 to avoid this.

Reply:
I guess that didn't work as we explicitly set C++17 in our CMakeLists.txt somewhere and override this, or the default GCC 11 doesn't have std::format. I think 13 is available...

Reply:
both are issues...

Guess I'll move to meson.

Reply:
the fmt wrap doesn't build on Cygwin. This will be fun to fix.

Reply:
Let's just install gcc13?

Reply:
I have merge access on WrapDB. I'll fix fmt first.

Reply:
oh there are even exiv2 bugs:

```
../src/helper_functions.cpp:15:31: error: 'strnlen' was not declared in this scope; did you mean 'strlen'?
   15 |   const size_t StringLength = strnlen(data, data_length);
```